geode-core/src/integrationTest/java/org/apache/geode/internal/jta/functional/CacheJUnitTest.java

```diff
@@ -454,7 +458,7 @@ public void testScenario5() throws Exception {
 
       current_fullpath = jtaObj.currRegion.getFullPath();
       assertEquals("failed retrieving current fullpath, current fullpath: " + current_fullpath,
-          "/" + DEFAULT_RGN + "/region1", current_fullpath);
+          SEPARATOR + DEFAULT_RGN + "/region1", current_fullpath);
```

Comment:
Did you miss this one?, shouldn't it be `SEPARATOR + DEFAULT_RGN + SEPARATOR + "region1"`?